lib/hosted_services.rb

```diff
@@ -1,5 +1,5 @@
 class HostedServices < ItemList
```

Comment:
You don't need this class at all.

The only difference between this and `ItemList` is that your todo_count (which you don't need, if you're not including this in the dashboard), returns `list.size` But, that is exactly the same as `list.length` which `ItemList#todo_count` returns.

So, you can just use `ItemList` for this, and remove the `HostedServices` class altogether.